sktime/transformers/rocket.py

```diff
@@ -0,0 +1,240 @@
+from numba import njit, prange
+import numpy as np
+import pandas as pd
+
+from sklearn.utils.validation import check_random_state
+from sktime.utils.validation import check_is_fitted
+
+from sktime.transformers.base import BaseTransformer
+from sktime.utils.data_container import dataframe_to_numpy
+from sktime.utils.validation.supervised import validate_X
+
+__author__ = "Angus Dempster"
+__all__ = ["ROCKET"]
+
+class Rocket(BaseTransformer):
+    """ROCKET
+
+    RandOm Convolutional KErnel Transform
+
+    @article{dempster_etal_2019,
+      author  = {Dempster, Angus and Petitjean, Fran\c{c}ois and Webb, Geoffrey I},
+      title   = {ROCKET: Exceptionally fast and accurate time series classification using random convolutional kernels},
+      year    = {2019},
+      journal = {arXiv:1910.13051}
+    }
+
+    Parameters
+    ----------
+    num_kernels : int, number of random convolutional kernels
+    seed        : int, random seed (optional, default None)
+    """
+
+    def __init__(self, num_kernels = 10_000, seed = None):

```

Comment:
Please rename `seed` to `random_state` to have common names!

Reply:
Done, but see comment below.